src/index.html

```diff
@@ -13,6 +13,38 @@
     />
```

Comment:
The stylesheet link references a `.scss` file. Ensure that this file is being compiled to a `.css` file and that the correct path is used here. Browsers do not directly interpret `.scss` files.